Assets/Scripts/UI/InGame/Graphics/GeneralGraphics.cs

```diff
@@ -1,23 +1,33 @@
 ﻿using System;
 using UnityEngine;
 using UnityEngine.UI;
+using Assets.Scripts.Graphics;
 
 namespace Assets.Scripts.UI.InGame
 {
     public class GeneralGraphics : MonoBehaviour
     {
+        [SerializeField] private Dropdown textureQuality;
+        [SerializeField] private Dropdown shadowRes;
+        [SerializeField] private Dropdown antiAliasing;
+        [SerializeField] private Dropdown shadows;
+        [SerializeField] private Toggle vSync;
+        [SerializeField] private Toggle softParticles;
+        [SerializeField] private Toggle customSettings;
+        [SerializeField] private QualitySwitcher qualitySwitcher;
+        [SerializeField] private InputField fpsLimit;
 
-        public Text VSyncCheck;
-
-        [SerializeField] public Dropdown textureQuality;
-        [SerializeField] public Dropdown shadowRes;
-        [SerializeField] public Dropdown antiAliasing;
-        [SerializeField] public Dropdown shadows;
-
-        [SerializeField] public Toggle vSync;
-        [SerializeField] public Toggle softParticles;
-        [SerializeField] public Toggle customSettings;
+        public InputField FPSLimit
+        {
+            get { return fpsLimit; }
+            set { fpsLimit = value; }
+        }
 
+        public QualitySwitcher QualitySwitcher
+        {
+            get { return qualitySwitcher; }
+            set { qualitySwitcher = value; }
+        }
```

Comment:
Rather make the fpsLimit & qualitySwitcher fields public, as the getter & setter aren't doing anything special here

Reply:
Although, looking at the other fields, it's also done there for some reason? So if you can, please also remove the useless properties from line 32 - 67 (TextureQuality till CustomSettings)

Reply:
i'll get on that, i'm a bit rusty so I just did it the way it had been done previously